keyboards/bm60rgb_iso/keymaps/iso_nordic_sleepmode/keymap.c

```diff
@@ -0,0 +1,120 @@
+     * | TAB | Q | W | E | R | T | Y | U | I | O | P | Å | ¨ | RET |
+     * |------------------------------------------------------\    |
+     * | CAPS | A | S | D | F | G | H | J | K | L | Ø | Æ | @ |    |
+     * |-----------------------------------------------------------|
+     * | ^  | < | Z | X | C | V | B | N | M | , | . |  ^/- | U | - |
+     * |-----------------------------------------------------------|
+     * |ctrl| win| alt|                        |alt| fn| L | D | R |
+     * `-----------------------------------------------------------' */
+    [0] = LAYOUT(
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,       KC_BSPC,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC,
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_BSLS,          KC_ENT,
+        KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  MT(MOD_RSFT, KC_SLSH),   KC_UP,  KC_SLSH,
+        KC_LCTL, KC_LGUI, KC_LALT,                  KC_SPC,                              KC_RALT, OSL(1),                KC_LEFT, KC_DOWN, KC_RGHT
+    ),
+    /* ,-----------------------------------------------------------.
+     * |GRA| f1| f2| f3| f4| f5| f6| f7| f8| f9|f10|f11|f12|  DEL  |
+     * |-----------------------------------------------------------|
+     * |     | R | R | R | R | R | R | R | R |   |   |   |   |     |
+     * |------------------------------------------------------\    |
+     * |      |   |   |   |   |   | H |PGD|PGU| E |   |   |   |    |
+     * |-----------------------------------------------------------|
+     * |    |   |   |   |   |   |   |   |   |   |   |      |V_D|V_U|
+     * |-----------------------------------------------------------|
+     * |    |    |    |                            |   |PRE|PLA|NEX|
+     * `-----------------------------------------------------------' */
+    [1] = LAYOUT(
+        KC_GRAVE,   KC_F1,   KC_F2,     KC_F3, KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,  KC_F10,  KC_F11,  KC_F12, KC_DEL,
+         _______, RGB_TOG, RGB_MOD, RGB_HUI, RGB_HUD, RGB_SAI, RGB_SAD, RGB_VAI, RGB_VAD, _______, _______, _______, _______,
+         _______, _______, _______, _______, _______, _______, KC_HOME, KC_PGDN, KC_PGUP,  KC_END, _______, _______, _______, _______,
+         _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_VOLD, KC_VOLU,
+         _______, _______, _______,                      _______,                         _______, _______, KC_MEDIA_PREV_TRACK, KC_MEDIA_PLAY_PAUSE, KC_MEDIA_NEXT_TRACK
+    ),
+};
+
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    #ifdef RGB_MATRIX_ENABLE
+    #ifdef SLEEPMODE_ENABLE
```

Comment:
This is a bit less verbose, easier to read, and formatted correctly. 
```suggestion
#if defined(RGB_MATRIX_ENABLE) && defined(SLEEPMODE_ENABLE)
```